migrations/20190704002811-create-mealfood.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('mealfoods', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      mealId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'meals',
+          key: 'id',
+          // This declares when to check the foreign key constraint. PostgreSQL only.
+          deferrable: Sequelize.Deferrable.INITIALLY_IMMEDIATE
+        }
+      },
+      foodId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'food',
+          key: 'id',
+          // This declares when to check the foreign key constraint. PostgreSQL only.
+          deferrable: Sequelize.Deferrable.INITIALLY_IMMEDIATE
```

Comment:
Probably want to research how this works in a more in-depth spike later down the road.